src/python/deepgnn/pytorch/modeling/base_model.py

```diff
@@ -121,9 +117,7 @@ def encode_feature(self, context: dict):
         if self.feature_enc:
             self.feature_enc.forward(context)
 
-    def forward(
-        self, context: QueryOutput
-    ) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+    def forward(self, context: QueryOutput) -> torch.Tensor:
```

Comment:
That's actually a big change, because I assumed all models inherit from base model. Should we just remove it and rely on `nn.Module.forward`?